app/models/bitflyer_connection.rb

```diff
@@ -90,6 +94,11 @@ def refresh_authorization!
     end
   end
 
+  # this is a hash of the account_id
+  def wallet_provider_id
```

Comment:
Add these methods for consistency, and rely on the same method call to determine if the passed file's provider ID matches ours.